Include accounts in wallet-redux

Bettelstab opened the pull request:
Belongs to https://github.com/nimiq/safe/pull/83

Bettelstab committed:
include accounts in wallet-redux

sisou reviewed (changes requested):
The transactions reducer _does not_ currently react to `LOGOUT_LEGACY` and `REMOVE_ACCOUNT` events, so transactions that belong to addresses deleted in those events are currently not removed, I think.

Bettelstab committed:
address review

transaction store: remove unused action, handle wallet/remove-account;
fix transaction rendering
